vms/platformvm/state/disk_staker_diff_iterator.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package state
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+const (
+	// startDiffKey = [subnetID] + [inverseHeight]
+	startDiffKeyLength = ids.IDLen + database.Uint64Size
+	// diffKey = [subnetID] + [inverseHeight] + [nodeID]
+	diffKeyLength       = startDiffKeyLength + ids.NodeIDLen
+	diffKeyHeightOffset = ids.IDLen
```

Comment:
I feel like using this constant is actually more confusing than just using `ids.IDLen` since the only time we use it is after copying a `subnetID`. I feel like it's more straightforward to be like, "we just copied an ID, so the offset is the length of an ID" in the code instead of "we just copied an ID, so the offset is a `diffKeyHeightOffset'"

Reply:
removed the constant